### PR TITLE
feat: add config for 'session_checks_intervall'

### PR DESCRIPTION
**Pull request by sebbousquet**

### CODE REVIEW

File: projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts [hunk 1]
```diff
@@ -255,6 +255,7 @@ export class AuthentificationService extends ArlasAuthentificationService {
           authConfigValue['silent_refresh_redirect_uri'] : url + 'silent-refresh.html',
         timeoutFactor: authConfigValue['timeout_factor'] !== undefined ? authConfigValue['timeout_factor'] : 0.75,
         sessionChecksEnabled: authConfigValue['session_checks_enabled'] !== undefined ? authConfigValue['session_checks_enabled'] : true,
+        sessionCheckIntervall: authConfigValue['session_checks_intervall'] !== undefined ? authConfigValue['session_checks_intervall'] : 3000,
```
Comment (MohamedHamouGisaia) [inline projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts:258]: interaval

Comment (sebbousquet) [reply projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts:258]: no, the config is `intervall` with 2 L

Comment (sebbousquet) [reply projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts:258]: https://manfredsteyer.github.io/angular-oauth2-oidc/docs/classes/AuthConfig.html#sessionCheckIntervall

**Review by MohamedHamouGisaia — APPROVED** [review]

File: projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/src/lib/services/authentification/authentification.service.ts:258]: my bad.

**Pull request merged**